test/includes.cnf

```diff
@@ -0,0 +1,5 @@
+#
```

Comment:
Some would argue that this kind of stuff belongs in dedicated test-specific subdirectory in test/recipes. For example there is 04-test_pem_data, 10-test_bn_data, etc.

Reply:
OK, I can move it there.